src/interface/mesh_data.hpp

```diff
@@ -406,92 +354,49 @@ class MeshData {
     return PackVariablesAndFluxesImpl(nullptr, nullptr, sparse_ids);
   }
   // No nothing
-  const auto &PackVariablesAndFluxes(PackIndexMap &map, vpack_types::StringPair &key) {
-    return PackVariablesAndFluxesImpl(&map, &key);
-  }
-  const auto &PackVariablesAndFluxes(vpack_types::StringPair &key) {
-    return PackVariablesAndFluxesImpl(nullptr, &key);
-  }
   const auto &PackVariablesAndFluxes(PackIndexMap &map) {
     return PackVariablesAndFluxesImpl(&map, nullptr);
   }
   const auto &PackVariablesAndFluxes() {
     return PackVariablesAndFluxesImpl(nullptr, nullptr);
   }
 
-  // As above, no variatics because they shadow each other.
+  // As above, NO TEMPLATE ARGUMENTS, as they shadow each other
   // Now the pack variables overloads.
   // covers names and metadata flags
   template <typename Elem>
-  const auto &PackVariables(const std::vector<Elem> names_or_flags,
-                            const std::vector<int> &sparse_ids, PackIndexMap &map,
-                            std::vector<std::string> &key, bool coarse = false) {
-    return PackVariablesImpl(&map, &key, coarse, names_or_flags, sparse_ids);
-  }
-  template <typename Elem>
   const auto &PackVariables(const std::vector<Elem> names_or_flags,
                             const std::vector<int> &sparse_ids, PackIndexMap &map,
                             bool coarse = false) {
     return PackVariablesImpl(&map, nullptr, coarse, names_or_flags, sparse_ids);
```

Comment:
Why are we keeping the `key_out` function argument in `PackVariablesImpl`? It looks like we're always passing `nullptr` now, so we might as well remove it, right? Same goes for `PackVariablesAndFluxes`.

Reply:
Ah good point. I'll remove it.